File: Back/src/BarberShop/Models/Cliente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;

namespace BarberShop.Models
{
    public class Cliente : Pessoa 
    {
        public string Email { get; set; }
       
        public string Senha { get; set; }

        public IEnumerable<Agenda> Agendas { get; set; }

        public IEnumerable<ClienteProfissional> ClientesProfissionais { get; set; }
    }
}

File: Front/ProjetoAPI/src/app/models/Cliente.ts
import { Agenda } from "./Agenda";
import { Profissional } from "./Profissional";

export interface Cliente {
  id: number;
  nome: string;
  telefone: string;
  email: string;
  senha: string;
  agendas:Agenda[];
  clientesProfissionais: Profissional[];
}
